app/client/components/dashboard/usagePiechart.jsx

```diff
@@ -1,102 +1,64 @@
 import React from 'react';
 import { connect } from 'react-redux';
-
 import { Pie } from 'react-chartjs-2';
-
-import { logout } from '../../reducers/auth/actions'
-
 import {ModuleHeader} from '../styledElements.js'
 
-import LoadingSpinner from "../loadingSpinner.jsx";
-
 const mapStateToProps = (state) => {
   return {
-    creditTransfers: Object.keys(state.creditTransfers.byId).map(id => state.creditTransfers.byId[id])
-  };
-};
-
-const mapDispatchToProps = (dispatch) => {
-  return {
-    logout:       () => dispatch(logout())
+    creditTransferStats: state.metrics.metricsState,
```

Comment:
make pie chart metrics dependent on entire record of credits not currently fetched